pkg/cmd/run.go

```diff
@@ -138,6 +138,12 @@ doppler run --mount secrets.json -- cat secrets.json`,
 		// retrieve secrets
 		dopplerSecrets := fetchSecrets(localConfig, enableCache, fallbackOpts, metadataPath, nameTransformer, dynamicSecretsTTL)
 
+		if !cmd.Flag("mount").Changed {
+			err := utils.CheckForDangerousSecretNames(dopplerSecrets)
```

Comment:
A comment noting why this is only done when not mounting would be helpful

Reply:
What is the reason we don't want to check for dangerous secret names when mounting the secrets in an ephemeral file?

Reply:
These are only "dangerous" because they can alter the shell's/interpreter's behavior. If we're mounting an ephemeral file, we're not adding these secrets as environment variables, so there's no risk.

Reply:
That makes 100% sense! Thank you